diff/diff_helpers_test.go

```diff
@@ -145,3 +146,35 @@ bar
 		})
 	}
 }
+
+func Test_MaskEnvVarsValues(t *testing.T) {
+	tests := []struct {
+		name    string
+		args    string
+		want    string
+		envVars map[string]string
+	}{
+		{
+			name: "JSON",
```

Comment:
Can you please add a test cases making sure that:
1. an unused env var doesn't result in any masked field (like having `DECK_FOO=foo` with no `foo` field existing)
2. an env var only partially matching a value doesn't result in any masked field (like having `DECK_FOO=foo` with some value being `foo_test`)
3. an env var used multiple times results in multiple masked values

Reply:
1 and 2 won't work as you mentioned because of the implementation:

```golang
func maskEnvVarValue(diffString string) string {
	for _, envVar := range parseDeckEnvVars() {
		diffString = strings.Replace(diffString, envVar.Value, "[masked]", -1)
	}
	return diffString
}
```

This is the same behavior of Jenkins or Gitlab uses IIRC.

3 will work and I can implement it. 

Just let me know if you think 1 & 2 is a blocker

Reply:
Yeah I think something like this would be good enough:

```
		{
			name: "JSON",
			envVars: map[string]string{
				"DECK_BARR": "barbar",
				"DECK_BAZZ": "bazbaz",   // used more than once
				"DECK_FUB":  "fubfub",   // unused
				"DECK_FOO":  "foo_test", // unused, partial match
			},
			args: `{"foo":"foo","bar":"barbar","baz":"bazbaz","another":"bazbaz"}`,
			want: `{"foo":"foo","bar":"[masked]","baz":"[masked]","another":"[masked]"}`,
		},
```

Other than that, this look great already!

Reply:
okay! I added tests for this scenario